docs/usage/tool_params.md

```diff
@@ -52,7 +52,16 @@ Seqtk is used for both the sub-sampling of reads and conversion of fasta files t
     - seed: A seed value for sub-sampling
     - reads_ext: Extension of reads after sub-sampling, do not touch alter this unless doing pipeline development.
     - assembly_fastq: Extension of the fastas after being converted to fastq files. Do no touch this unless doing pipeline development.
-    - report_tag: Name of seqtk data in the final summary report. Do no touch this unless doing pipeline development.
+    - report_tag: Name of seqtk data in the fi nal summary report. Do no touch this unless doing pipeline development.
```

Comment:
space in final

Reply:
fixed in: https://github.com/phac-nml/mikrokondo/pull/125/commits/667b67d77d74bffb3d9ccedde33cd0f95ee0a188